setup/declare.sh

```diff
@@ -0,0 +1,2 @@
+# Run this the first time only
+eups declare -r . Twinkles $USER  --tag=$USER
```

Comment:
This is going to fall prey to the bug outlined here

https://community.lsst.org/t/change-of-behavior-in-eups/523/7

safer to do
```
eups declare -r . Twinkles $USER
eups declare -r . Twinkles $USER --tag=$USER
````
as Russel explains in that community post (otherwise, on EUPS 2.0.1, the tag will get blown away every time EUPS is re-setup; this is a bug that will be fixed in the next EUPS release).